src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPaneEditorStyles.css

```diff
@@ -183,5 +185,20 @@ p + div[style="text-align: center;"]
    color: inherit !important;
 }
 
+/**
+ * This styling ensures the sourceCode area has a solid, continuous
+ * background color, even if the view is narrowed and a scrollbar
+ * appears to allow for horizontal scrolling.
+ */
+ .ace_editor_theme div.sourceCode {
+   border-radius: 3px;
+}
+
+.rstudio-themes-default.ace_editor_theme div.sourceCode {
```

Comment:
This has a slightly different look in light mode for some editor themes. When switching to `Chrome` or `Eclipse` for example, it doesn't cover the entire background with the colour. It seems that there's no `rstudio-themes-default` for these themes but there is a `rstudio-themes-light` class on it.

![image](https://user-images.githubusercontent.com/9591545/213570340-093bac1b-513c-4249-8589-9392804af2a3.png)

Reply:
For some reason `rstudio-themes-default` shows up for the themes `Chrome` and `Eclipse` on my end (and the background colour is covered), but I can use `rstudio-themes-light-menus` instead which also works for light themes on my end.

I've also changed the dark theme rule to use `rstudio-themes-dark-menus` instead, so it's similar to the light theme rule. I tested with various light and dark themes and seems to be working for me. Let me know if you see different behaviour!

Reply:
I tried out some different settings and `rstudio-themes-default` is tied to the Appearance -> RStudio Theme setting with the `Modern` value. Changing it to `Sky` will use `rstudio-themes-alternate`.

It feels kind of weird to use `rstudio-themes-[light|dark]-menus` for the selector. I think a CSS OR selector would work better here to handle the different classes used to represent `Modern` and `Sky`.